_posts/en/newsletters/2021-11-17-newsletter.md

```diff
@@ -0,0 +1,154 @@
+---
+title: 'Bitcoin Optech Newsletter #175'
+permalink: /en/newsletters/2021/11/17/
+name: 2021-11-17-newsletter
+slug: 2021-11-17-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter provides information about the activation of
+taproot and includes our regular sections with summaries of changes to
+services and client software, new releases and release candidates, and
+notable changes to popular Bitcoin infrastructure software.
+
+## News
+
+- **Taproot activated:** as expected, the [taproot][topic taproot] soft
+  fork activated at block height {{site.trb}}.  As of this writing,
+  several large mining pools are not mining blocks containing taproot
+  spends.  This may indicate that they were falsely signaling readiness
+  to enforce taproot's rules, a risk we [previously warned about][p4tr
+  what happens].  Alternatively, they may be risklessly using a
+  taproot-enforcing node to choose which block chain to use while also
+  using an older node or custom software to choose which transactions to
+  include in their blocks.
+
+    The safest course of action for users and businesses is to run their
+    own taproot-enforcing node (such as Bitcoin Core 22.0) and only
+    accept transactions confirmed by it.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **bitcoinj adds bech32m, P2TR support:**
+  Andreas Schildbach added [a commit][bitcoinj bech32m] for [bech32m][topic
+  bech32] and another for [P2TR support][bitcoinj p2tr] to the bitcoinj repository.
+
+- **libwally-core adds bech32m support:**
+  The [0.8.4 release][libwally 0.8.4] of this wallet primative library includes [bech32m
+  support][libwally 297].
+
+- **Spark Lightning Wallet adds BOLT12 offers:**
+  Spark [v0.3.0][spark v0.3.0] adds [offer][topic offers] features including
+  offer creation, sending offer payments, and pull payments. Recurring offer
+  features are planned for a future release.
+
+- **BitGo wallets support taproot:**
+  BitGo [announced][bitgo taproot blog] support for both sending from and
+  receiving to [taproot][topic taproot] outputs using their API. Taproot support
+  in the UI is planned for a future update.
+
+- **NthKey adds bech32m send capabilities:**
+  iOS signing service [NthKey][nthkey website] added support for taproot sends
+  in the [v1.0.4 release][nthkey v1.0.4].
+
+- **Ledger Live supports taproot:**
+  Ledger's client software, Ledger Live, announced taproot support in their
+  [v2.35.0 release][ledger v2.35.0] as an experimental feature.
+
+- **Muun wallet supports taproot:**
+  Muun wallet enabled taproot address support after activation occurred,
+  including the ability for users to default to taproot receive addresses.
```

Comment:
@bitschmidty Is it deliberate that there's no link for the Muun item?  It's fine if deliberate (e.g. there's no good link).

Reply:
Yes, there wasn’t a good link but lots of twitter screenshots only.